clean_project/construct_dict.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+Construct a JSON file used to indexing all words in a config_file.yaml
+"""
+import argparse
+import codecs
+import json
+import os
+import yaml
+
+from collections import defaultdict
+from clean_file import sanitize
+
+
+def construct_dictionary(yaml_path, path_json_file):
+    """
+    Construct the dictionary, given a configuration files
+    :param yaml_path: Yaml's path that contains a list of files that will be cleaned
+    :param path_json_file: Path of JSON file (indexing)
```

Comment:
nit: `path_json` would sound better as `json_path`. also: what are you serializing in json? do you really need to be so abstract? many things can be json. this would be better be called as `index_path`